js/index.js

```diff
@@ -1 +1,119 @@
 // Your code goes here
+
+////////////// MOUSEOVER 
+const navChange = document.querySelector('.nav');
+
+navChange.addEventListener('mouseover', function(e) {
+    e.target.style.color = 'orange';
+
+    setTimeout(function() {
+        e.target.style.color = 'black';
+      }, 500);
+    }, false);
+
+
+
+////////////// KEYUP  + KEYDOWN
+function getRandomColor() {
```

Comment:
Great job using a function  that uses a for loop and math.floor and math.random to create random colors